src/main/webapp/app/entities/administrator/administrator-update.tsx

```diff
@@ -81,25 +104,17 @@ export class AdministratorUpdate extends React.Component<IAdministratorUpdatePro
               <p>Loading...</p>
             ) : (
               <AvForm model={isNew ? {} : administratorEntity} onSubmit={this.saveEntity}>
-                {!isNew ? (
-                  <AvGroup>
-                    <Label for="administrator-id">
-                      <Translate contentKey="global.field.id">ID</Translate>
-                    </Label>
-                    <AvInput id="administrator-id" type="text" className="form-control" name="id" required readOnly />
-                  </AvGroup>
-                ) : null}
                 <AvGroup>
-                  <Label id="userIdLabel" for="administrator-userId">
-                    <Translate contentKey="clubmanagementApp.administrator.userId">User Id</Translate>
+                  <Label for="user-id">
+                    <Translate contentKey="global.form.username.label">User Name</Translate>
                   </Label>
-                  <AvField id="administrator-userId" type="string" className="form-control" name="userId" />
-                </AvGroup>
-                <AvGroup>
-                  <Label id="yearSessionLabel" for="administrator-yearSession">
-                    <Translate contentKey="clubmanagementApp.administrator.yearSession">Year Session</Translate>
-                  </Label>
-                  <AvField id="administrator-yearSession" type="text" name="yearSession" />
+                  <AvSelectField
+                    name="userId"
+                    options={selectOptions}
+                    validate={{
+                      required: { value: true, errorMessage: 'Please select user name' },
+                    }}
+                  />
```

Comment:
for create administrator this is good
but when update the user name should be readonly

Reply:
resolved by https://github.com/Five-Fishes/Club-Management-React-Client/pull/156/commits/94c83b6bfa3404d4bbc3c3fa0d1f7ebfec0137e2